pkgs/by-name/ai/aicommit2/package.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nodejs,
+  pnpm,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "aicommit2";
+  version = "2.1.5";
+
+  src = fetchFromGitHub {
+    owner = "tak-bro";
+    repo = "aicommit2";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-/bl0Qo4y3rDdGXIKTdjC5wtbeW1NUaIMkw5uzSQ+8Dk=";
+  };
+
+  pnpmDeps = pnpm.fetchDeps {
+    inherit (finalAttrs) pname version src;
+    hash = "sha256-y3IftcRhsDDyMZ2AE07WyY94mp0j6M6UVno7Qp8d/G8=";
+  };
+
+  nativeBuildInputs = [
+    nodejs
+    pnpm.configHook
+  ];
+  buildInputs = [ nodejs ];
+
+  buildPhase = ''
+    runHook preBuild
+
+    pnpm build
+
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/{bin,lib/aicommit2}
+    cp -r {dist,node_modules} $out/lib/aicommit2
+
+    runHook postInstall
+  '';
+
+  preFixup = ''
+    patchShebangs $out/lib/aicommit2/dist/cli.mjs
+    ln -s $out/lib/aicommit2/dist/cli.mjs $out/bin/aicommit2
+    ln -s $out/lib/aicommit2/dist/cli.mjs $out/bin/aic2
+  '';
```

Comment:
patchShebangs automatically happen and works well.
If you want you can remove the patchShebangs line and move the symlinks to InstallPhase,
and then remove the preFixup phase....

Reply:
done